migration/20190320-1-add-license-to-loans.sql

```diff
@@ -0,0 +1,2 @@
+alter table loans add column license_id integer;
+alter table loans add constraint loans_license_id_key foreign key (license_id) references licenses(id);
```

Comment:
These migrations should be in core (even though circ is the only component that uses these fields) because the model code is also in core.